GitHubVS.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.27130.0
```

Comment:
I'm not sure about this change.

Reply:
This change would break me. I have too many versions of VS 2017 installed. ;-) Can you revert it?